style.css

```diff
@@ -0,0 +1,4 @@
+h1 {
+  background-color: aqua;
+  font-family: monospace;
+}
```

Comment:
- [ ] Instead of targeting the h1 element, what do you think about creating a class instead and assigning it to the header tag or element as instructed in the project guidelines. 🤗

Reply:
OMG, thanks a lot for your suggestion. 
Of course, I must do it, it is in the project guidelines! 🙌   🙌